### PR TITLE
VcenterPropertiesCollector

### PR DESCRIPTION
**Pull request by durcar86**
Hello, here is a vCenter properties Collector to get Build version from vRops

vrops_vcenter_properties{datacenter="NAME",propkey="summary_version: 6.5.0",vcenter="FQDN"} 0.0
vrops_vcenter_properties{datacenter="NAME",propkey="vc_fullname: VMware vCenter Server 6.5.0",vcenter="FQDN"} 0.0

### CODE REVIEW

File: collectors/VCenterPropertiesCollector.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import os, time, json
+from prometheus_client.core import GaugeMetricFamily
+from tools.Resources import Resources
+from tools.YamlRead import YamlRead
+
+class VcenterPropertiesCollector(BaseCollector):
+    def __init__(self):
+        self.wait_for_inventory_data()
+        self.property_yaml = YamlRead('collectors/property.yaml').run()
+        self.g = GaugeMetricFamily('vrops_vcenter_properties', 'testtest',
```
Comment (viennaa) [inline collectors/VCenterPropertiesCollector.py:11]: please configure this in collect(), have a glimpse at the other modules, this has changed

File: collectors/VCenterPropertiesCollector.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                              labels=['datacenter', 'vcenter', 'propkey'])
+        self.post_registered_collector(self.__class__.__name__, self.g.name)
+
+    def describe(self):
+        yield self.g
```
Comment (viennaa) [inline collectors/VCenterPropertiesCollector.py:16]: yield the GaugeMetricFamily straight, look at the other collectors

File: collectors/VCenterPropertiesCollector.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            print('vCenterPropertiesCollector starts with collecting the properties')
+
+        for vc in self.get_vcenters():
+            target = self.vcenters[vc]['target']
+            target_vc = self.vcenters[vc]['name']
+            token = self.vcenters[vc]['token']
+            vc_uuid = self.vcenters[vc]['uuid']
+            for cl in self.get_clusters():
+                if self.clusters[cl]['vcenter'] == target_vc:
+                    dc_name = self.clusters[cl]['parent_dc_name']
```
Comment (viennaa) [inline collectors/VCenterPropertiesCollector.py:29]: we don't need dc_name. please also remove dc from the metric itself

File: collectors/VCenterPropertiesCollector.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+            if not token:
+                print("skipping " + target + " in vCenterPropertiesCollector, no token")
+
+            for property_pair in self.property_yaml["VcenterPropertiesCollector"]['info_metrics']:
```
Comment (viennaa) [inline collectors/VCenterPropertiesCollector.py:34]: please check, if the info_metrics key is present in property_yaml.

File: exporter.py [hunk 5]
```diff
@@ -76,7 +77,8 @@ def run_prometheus_server(port, collectors,  *args):
                 HostSystemPropertiesCollector(),
                 DatastoreStatsCollector(),
                 VMStatsCollector(),
-                VMPropertiesCollector()
+                VMPropertiesCollector(),
+                VcenterPropertiesCollector()
```
Comment (viennaa) [inline exporter.py:81]: Please rename to `VCenterPropertiesCollector`

File: tests/TestCollectors.py [hunk 6]
```diff
@@ -23,6 +23,8 @@
 from collectors.VMStatsCollector import VMStatsCollector
 from collectors.VMPropertiesCollector import VMPropertiesCollector
 from collectors.ClusterStatsCollector import ClusterStatsCollector
+from collectors.VCenterPropertiesCollector import VcenterPropertiesCollector
+
```
Comment (viennaa) [inline tests/TestCollectors.py:27]: please remove empty line

File: collectors/VCenterPropertiesCollector.py (hunk 4 above)
Comment (durcar86) [reply collectors/VCenterPropertiesCollector.py:34]: info_metrics was added to the property.yaml 

https://github.com/durcar86/vrops-exporter/blob/VCenterProperties/collectors/property.yaml

File: collectors/VCenterPropertiesCollector.py (hunk 1 above)
Comment (durcar86) [reply collectors/VCenterPropertiesCollector.py:11]: not sure about this, let me check and get back to you

**Pull request closed without merging**